### DISCUSSION TITLE
My account has been blocked

### DISCUSSION THREAD

**Discussion by vighnesh153** [vercel/community#3373] [Help] [answered]
### Summary

My account has been blocked and I didn't even get an email about the reason for the blocking. I have a bunch of services on Vercel and I am not able to access them. 

I can't even contact support because I need to sign in to my account to get in touch with the support.

Can I at least get the reason behind this?

### Example

https://vighnesh153.com

### Steps to Reproduce

Visit 

https://vighnesh153.com

https://tsx.vighnesh153.com

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.

**Comment by vighnesh153** [comment]
Thanks. LOVE THIS AWESOME SERVICE BY VERCEL 🦄 It made me accelerate my migration to AWS ❤️
